bentoml/_internal/frameworks/transformers.py

```diff
@@ -70,42 +75,47 @@ def _check_flax_supported() -> None:  # pragma: no cover
             )
 
 
-try:
+if TYPE_CHECKING:
     import transformers
-
-    _check_flax_supported()
-except ImportError:  # pragma: no cover
-    raise MissingDependencyException(
-        """\
-        transformers is required in order to use module `bentoml.transformers`.
-        Instruction: Install transformers with `pip install transformers`.
-        """
+    import transformers.utils.import_utils as import_utils
+else:
+    _exc_msg = "transformers is required in order to use module 'bentoml.transformers'. Install transformers with 'pip install transformers'."
+    transformers = LazyLoader(
+        "transformers", globals(), "transformers", exc_msg=_exc_msg
+    )
+    import_utils = LazyLoader(
+        "import_utils", globals(), "transformers.utils.import_utils"
     )
+    _check_flax_supported()
```

Comment:
provide lazy loading.

Reply:
no need for lazy loading under type checking

Reply:
solved